### DISCUSSION TITLE
Xamarin Android

### DISCUSSION THREAD

**Discussion by Pawel-VRtechnology** [xamarin/TorchSharp#1083] [General]
Hello everyone is it possible use TorchSharp in Maui android platform ?

**Comment by jck-chan** [comment]
But it seems that TorchSharp currently only supports CUDA and many Android phones don't support CUDA GPU.

**Comment by dayo05** [comment]
It depends on Libtorch(PyTorch's native library). If does, then you can manually compile it as M1 MacOS does even if it doesn't supported officially.

IMO, because PyTorch is very huge library so I suggest not to use on device even if that supports. As alternative, you can send data to server and process it with this library there.
